mappings/com/mojang/blaze3d/platform/GLX.mapping

```diff
@@ -0,0 +1,218 @@
+	FIELD field_30545 isOpenGl21 Z
+	FIELD field_30546 usePostProcess Z
+	FIELD field_30547 useVbo Z
+	FIELD field_30548 isAmdVboSupported Z
+	FIELD field_30549 arrayBuffer I
+	FIELD field_30551 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_30552 fboMode Lnet/minecraft/class_6125$class_6126;
+	FIELD field_30553 hasShaders Z
+	FIELD field_30554 useShaderArb Z
+	FIELD field_30555 useMultitextureArb Z
+	FIELD field_30556 useTexEnvCombineArb Z
+	FIELD field_30557 separateBlend Z
+		COMMENT States whether OpenGL 1.4 features are supported.
+		COMMENT
+		COMMENT <p>
+		COMMENT \tThis includes
+		COMMENT \t<ul>
+		COMMENT \t\t<li>Separate Blend Functions</li>
+		COMMENT \t\t<li>Automatic mipmap generation</li>
+		COMMENT \t\t<li>Stencil wrapping</li>
+		COMMENT \t</ul>
+		COMMENT \tand many more.
+		COMMENT </p>
+	FIELD field_30558 isNvidia Z
+	FIELD field_30559 capsString Ljava/lang/String;
+	FIELD field_30560 cpuInfo Ljava/lang/String;
+	FIELD field_30561 useVboArb Z
+	FIELD field_30562 isAmd Z
+	FIELD field_30563 framebuffer I
+	FIELD field_30564 renderbuffer I
+	FIELD field_30565 colorAttachment I
+	FIELD field_30566 depthAttachment I
+	FIELD field_30567 completeFramebuffer I
+	FIELD field_30568 incompleteFramebufferAttachment I
+	FIELD field_30569 incompleteFramebufferAttachmentMiss I
+	FIELD field_30570 incompleteFramebufferAttachmentDraw I
+	FIELD field_30571 incompleteFramebufferAttachmentRead I
+	FIELD field_30572 advancedOpenGlEnabled Z
+	FIELD field_30573 linkStatus I
+	FIELD field_30574 compileStatus I
+	FIELD field_30575 vertexShader I
+	FIELD field_30576 fragmentShader I
+	FIELD field_30577 textureUnit I
+	FIELD field_30578 lightmapTextureUnit I
+	FIELD field_30579 texture I
+	FIELD field_30580 combine I
+	FIELD field_30581 interpolate I
+	FIELD field_30582 primaryColor I
+	FIELD field_30583 constant I
+	FIELD field_30584 previous I
+	FIELD field_30585 combineRgb I
+	FIELD field_30586 source0Rgb I
+	METHOD method_31782 setupContext ()V
+	METHOD method_31783 deleteShader (I)V
+		COMMENT Deletes a shader object
+		ARG 0 shader
+	METHOD method_31784 multiTexCoord2f (IFF)V
+		COMMENT Sets the current texture coordinates
+	METHOD method_31785 getProgram (II)I
+		COMMENT Returns a parameter from a program object
+		ARG 0 program
+		ARG 1 param
+	METHOD method_31786 renderbufferStorage (IIII)V
+		COMMENT Establishes data storage, format and dimensions of a renderbuffer object's image
+	METHOD method_31787 framebufferTexture2DEXT (IIIII)V
+		COMMENT Attaches a level of a texture object as a logical buffer of a framebuffer object
+	METHOD method_31788 getUniformLocation (ILjava/lang/CharSequence;)I
+		COMMENT Returns the location of a uniform variable
+		ARG 0 program
+		ARG 1 name
+	METHOD method_31789 shaderSource (ILjava/nio/ByteBuffer;)V
+		COMMENT Replaces the source code in a shader object
+		ARG 0 shader
+		ARG 1 source
+	METHOD method_31790 bindBuffer (ILjava/nio/ByteBuffer;I)V
+		COMMENT Binds a named buffer object
+		ARG 1 buffer
```

Comment:
`bufferData`
Might need to change the javadoc too

Reply:
but why
these are param names
khronos calls it that

Reply:
khronos? And i meant the method name not the param.. i selected the whole thing cause you might need to change the javadoc too

Reply:
the method name matches lwjgl’s without the gl prefix
that’s definitely got to be higher than upstream in preference

Reply:
But you said we shouldnt match unobf names

Reply:
upstream is always priority

Reply:
wdym? gl methods’ names should match their lwjgl name

Reply:
Why does upstream call it that then?

Reply:
upstream is the highest priority.

Reply:
will not rename this
`buffer` and `bufferData` mean the exact same thing
khronos and lwjgl call it `buffer` in their natives and bindings respectively
it is not yarn's role to confuse devs by changing names of gl methods' dispatchers

Reply:
do not resolve this without waiting for other participants to respond. we have no say in what happens upstream yet we follow it. please change this name

Reply:
> we have no say in what happens upstream yet we follow it.

no

Reply:
![image](https://user-images.githubusercontent.com/62552372/105660178-79967e80-5ef0-11eb-8876-d52302405bbe.png)
do you see why I'm not renaming it?
we follow this convention everywhere

Reply:
oh wait

Reply:
nvm im stupid

Reply:
didnt read it

Reply:
false alarm